framework/doc/content/source/postprocessors/ThermocoupleSensorPostprocessor.md

```diff
@@ -0,0 +1,17 @@
+# ThermocoupleSensorPostprocessor
+
+## Syntax and Description
+
+!syntax description /Postprocessors/ThermocoupleSensorPostprocessor
+
+`ThermocoupleSensorPostprocessor` models very specific classes of thermocouples,
```

Comment:
I think this belongs in the misc module. 

unless you are using it in a particular application? if so which one?